tests/functionnal/CommonDBTM.php

```diff
@@ -114,4 +114,66 @@ public function testgetIndexNameOtherThanID() {
       ]))->isFalse();
 
    }
+
+   /**
+    * @engine inline
```

Comment:
Engine should not be required (I'd prefer avoid using this whan this is not required).

Reply:
sorry, this was only for debug purposes. Removing